### ISSUE TITLE
[BUG]: Version check (?) possibly causing error on Colab

### ISSUE THREAD

**Issue by sobotka** [colour-science/colour#1052]
### Description

Attempting an install from GitHub yields an error.

```
ERROR: Package 'colour' requires a different Python: 3.7.14 not in '<3.11,>=3.8'
```

[The error may be related to the syntax on the version check](https://github.com/python-poetry/poetry/issues/1413#issuecomment-557195997). Have tested with other various Pythons, and the error remains.

**Comment by KelSolaar** [comment]
Hello,

The minimum supported Python version is 3.8 if you want to install from Github, our How-To should show how to do it (at the top): https://colab.research.google.com/notebook#fileId=1NRcdXSCshivkwoU2nieCvC3y14fx1X4X&offline=true&sandboxMode=true

Let me know how it goes!

**Comment by sobotka** [comment]
It seems to be an error in parsing.

I fixed it by fixing this line:

https://github.com/sobotka/colour/blob/develop/pyproject.toml#L46

```
python = "> 3.7, <= 3.8"
```

The parser was showing the wrong greater than (Python versioning) direction, and when I changed it, it flaked out. So I thought the error could be the fact that `3.11` looks "greater than" the `3.8`, and changed it to `3.7` and fixed the comparison operators.

Solved the issue on Colab. Bit of a nightmare to fix on your end I suspect?
